google/cloud/bigtable/internal/bigtable_stub_factory_test.cc

```diff
@@ -259,15 +264,15 @@ TEST_F(BigtableStubFactory, AsyncMutateRow) {
 TEST_F(BigtableStubFactory, FeaturesFlags) {
   MockFactory factory;
   EXPECT_CALL(factory, Call)
-      .WillOnce([](std::shared_ptr<grpc::Channel> const&) {
+      .WillOnce([=](std::shared_ptr<grpc::Channel> const&) {
```

Comment:
What local objects are we wanting to capture?

Reply:
D'oh those were artifacts from a previous commit. Removed.